demisto_sdk/commands/common/tools.py

```diff
@@ -562,17 +567,25 @@ def get_remote_file(
     tag: str = "master",
     return_content: bool = False,
     git_content_config: Optional[GitContentConfig] = None,
+    default_value=None
 ):
     """
     Args:
         full_file_path:The full path of the file.
         tag: The branch name. default is 'master'
         return_content: Determines whether to return the file's raw content or the dict representation of it.
         git_content_config: The content config to take the file from
+        default_value: The default value to return when running sdk offline.
     Returns:
         The file content in the required format.
 
     """
+
+    if is_sdk_defined_working_offline():
+        if not default_value:
+            raise NoInternetConnectionException()
+        return default_value
```

Comment:
I think we should treat `None` as _do not return a default value, always raise the exception_. 
Let's add this logic, and document it in the docstrings.